docs/settingup.md

```diff
@@ -2379,6 +2380,21 @@ The following functions are supported:
     }
 }
 
+```
+
+```json
+{
+    "action": "ElasticCreateApp",
```

Comment:
please use non elastic action as example, have a branch removing all mention of elastic actions from docs and don't want new ones.